applications/nrf5340_audio/src/bluetooth/le_audio.c

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "le_audio.h"
+
+#include <zephyr/zbus/zbus.h>
+#include <zephyr/bluetooth/conn.h>
+
+#include "nrf5340_audio_common.h"
+#include "bt_mgmt.h"
+#include "bt_rend.h"
+#include "bt_content_ctrl.h"
+#include "broadcast_internal.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(le_audio, CONFIG_BLE_LOG_LEVEL);
+
+static enum le_audio_device_type device_type;
+
+/**
+ * @brief	Zbus listener to receive events from bt_mgmt
+ *
+ * @note	Will in most cases be called from BT_RX context,
+ *		so there should not be too much processing done here
+ */
+static void bt_mgmt_evt_handler(const struct zbus_channel *chan)
```

Comment:
It looks strange that the bt_mgmt event handler is located in le_audio.c which is not in the bt_management folder

Reply:
It is handling the events FROM bt_mgmt, same as button_evt_handler is located in streamctrl